src/middlewared/middlewared/plugins/iscsi_/alua.py

```diff
@@ -535,3 +535,49 @@ async def removed_target_extent(self, target_id, extent_id):
         # etc, but (currently) these don't work.  Therefore we'll use a sledgehammer
         await self.middleware.call('failover.call_remote', 'iscsi.target.logout_ha_target', [targetname])
         await self.middleware.call('failover.call_remote', 'service.reload', ['iscsitarget'])
+
+    async def has_active_jobs(self):
```

Comment:
This can be simplified by checking the `state` attribute of the job. i.e.
```
running_jobs = await self.middleware.call(
    'core.get_jobs', [
        ('method', 'in', [
            'iscsi.alua.active_elected',
            'iscsi.alua.activate_extents',
            'iscsi.alua.standby_after_start',
            'iscsi.alua.standby_reload',
            'iscsi.alua.standby_fix_cluster_mode',
        ]),
        ('state', '=', 'RUNNING'),
    ]
)
return bool(running_jobs)